graph/src/components/store/entity_cache.rs

```diff
@@ -170,7 +177,8 @@ impl EntityCache {
             }
             None => {
                 let value = self.schema.id_value(&key)?;
-                entity.set("id", value);
+                // unwrap: our AtomPool always has an id in it
+                entity.set("id", value).unwrap();
```

Comment:
could be useful to have a set_id fn that doesn't return an error? I think it would make it easier to use correctly

Reply:
I'll have another PR that gets rid of single-key changes to entities - they are only needed in tests. In general, an `Entity` is used to shuttle data between WASM and the store, and we don't really need to update an `Entity` once it's been constructed (this `id` setting logic then moves to `store_set` before the `Entity` is constructed) That has the nice side-effect that an `Entity` always has an `id` and `Entity.id()` can just return `String`